contrib/dbus-test-runner/template.py

```diff
@@ -0,0 +1,35 @@
+pkgname = "dbus-test-runner"
+pkgver = "19.04.0"
+pkgrel = 0
+build_style = "gnu_configure"
+configure_args = ["--disable-static"]
+make_cmd = "gmake"
+hostmakedepends = [
+    "autoconf",
+    "automake",
+    "gmake",
+    "intltool",
+    "libtool",
+    "pkgconf",
+]
+makedepends = [
+    "dbus",
+    "dbus-glib-devel",
```

Comment:
dbus-glib is seemingly not even used, you just need to patch it out of configure.ac